versions.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_version = ">= 0.12.9"
 
   required_providers {
-    aws        = ">= 2.52.0"
+    aws        = ">= 2.60.0"
```

Comment:
Why this increase? I don't see anything added in that version that's required to support features/fixes in the module.

Reply:
Since this is going to require a major version bump of the module anyway, I thought it would be okay to bump AWS provider version to the latest.